app/models/user.rb

```diff
@@ -56,16 +56,20 @@ class User < ApplicationRecord
   scope :search_by, ->(param) { search_by_name(param).or(search_by_email(param)) }
   scope :sorted_by_organisation_and_role, -> { joins(:organisation).order("organisations.name", role: :desc, name: :asc) }
 
-  def lettings_logs
-    if support?
+  def lettings_logs(created_by: false)
+    if created_by
+      LettingsLog.created_by(self)
+    elsif support?
```

Comment:
This wasn't immediately clear, especially as it's called with (created_by: true) my first thought was that the logs had created_by not nil.
Would it make sense to use `.filter_by_user` scope where needed instead of changing this method?

Reply:
Also, wouldn't we still want to filter by organisation? In case we allow users to change org in the future or something

Reply:
I've reverted this related to your first comment. I have asked Jemma about your second comment.

Reply:
This still takes in created_by, am I missing something?